webapp/src/pages/RegisterPage/RegisterPage.css

```diff
@@ -0,0 +1,72 @@
+.register {
+  margin: 0;
+  padding: var(--padding-top-bottom) var(--padding-sides);
+  height: 100vh;
+  min-height: 40rem;
+  overflow: scroll;
+  background-color: var(--color-background);
+  color: var(--color-light);
+  display: grid;
+  grid-template-columns: repeat(var(--column-number), 1fr);
+  grid-template-rows: auto 1fr;
```

Comment:
have you noticed that we have been building all pages with a column gap that you are nor having not here not in the login page? V.g:
![Screenshot from 2022-09-15 23-36-38](https://user-images.githubusercontent.com/2774420/190504159-85f15473-eb0d-4e19-bde5-7b8ef6048acf.png)
![Screenshot from 2022-09-15 23-35-38](https://user-images.githubusercontent.com/2774420/190504162-b43cd604-ab92-449c-9909-d3830de5605b.png)

Reply:
No, sorry, I'll fix it

Reply:
I'm sorry, I made a mess with your pr. I wanted to delete the screenshots I posted, and instead I removed the css file. Should be fixed now (I forced pushed your commit from before I created my stupid commit)

Reply:
Don't worry. But I don't know why it's happening. That gap is in the root.css, isn't it? Shouldn't it be there by deafult?

Reply:
Done